components/sysutils/chezmoi/patches/01-only-build-illumos.patch

```diff
@@ -0,0 +1,25 @@
+#This patch file exists because the source code's Makefile does not build for illumos by default
```

Comment:
Update the text as following.

`Chezmoi by default uses git to find the version it is built from. This does not work as we download a tarball copy from Gitlab and build inside oi-userland, thus we remove the version embedding here`

Reply:
Complete